contrib/network/connection_balance/dlb/source/connection_balancer_impl.h

```diff
@@ -49,6 +51,33 @@ class DlbBalancedConnectionHandlerImpl : public Envoy::Network::BalancedConnecti
   Envoy::Event::FileEventPtr dlb_event_;
 };
 
+// The dir should always be "/dev" in production.
+// For test it is a temporary directory.
+// Return Dlb device id, absl::nullopt means error.
+static absl::optional<uint> detectDlbDevice(const uint config_id, const std::string& dir) {
```

Comment:
nit: you can use `absl::StatusOr` as it allows you to specify the error.

Reply:
Sorry for slow response due to sickness, I will optimize it in the next PR.